reference/commands/graph/info.rst

```diff
@@ -130,5 +130,379 @@ The ``conan graph info`` command shows information about the dependency graph fo
+        2: zlib/1.2.13#13c96f538b52e1600c40b88994de240f
+    zlib/1.2.13#13c96f538b52e1600c40b88994de240f:
+      ref: zlib/1.2.13#13c96f538b52e1600c40b88994de240f
+      id: 2
+      recipe: Cache
+      package_id: 6fe7fa69f760aee504e0be85c12b2327c716f9e7
+      prev: None
+      build_id: None
+      binary: Skip
+      invalid_build: False
+      url: https://github.com/conan-io/conan-center-index
+      license: Zlib
+      description: A Massively Spiffy Yet Delicately Unobtrusive Compression Library (Also Free, Not to Mention Unencumbered by Patents)
+      topics: ('zlib', 'compression')
+      homepage: https://zlib.net
+      revision_mode: hash
+      package_type: static-library
+      settings:
+        os: Macos
+        arch: x86_64
+        compiler: apple-clang
+        compiler.version: 14
+        build_type: Release
+      system_requires:
+      options:
+        fPIC: True
+        shared: False
+      recipe_folder: /Users/barbarian/.conan2/p/zlib9f370ca971ddf/e
+      source_folder: None
+      build_folder: None
+      package_folder: None
+      cpp_info:
+        root:
+          includedirs: ['include']
+          srcdirs: None
+          libdirs: ['lib']
+          resdirs: None
+          bindirs: ['bin']
+          builddirs: None
+          frameworkdirs: None
+          system_libs: None
+          frameworks: None
+          libs: None
+          defines: None
+          cflags: None
+          cxxflags: None
+          sharedlinkflags: None
+          exelinkflags: None
+          objects: None
+          sysroot: None
+          requires: None
+          properties: None
+      label: zlib/1.2.13
+      context: host
+      test: False
+      requires:
+
+
+You can additionally filter the output, both by filtering by fields (``--filter``) and by package (``--filter-package``).
+For example, to get the options of zlib, the following command could be run:
+
+.. code-block:: text
+
+    $ conan graph info --require=binutils/2.38 -r=conancenter --filter=options --package-filter="zlib*"
+
+    ...
+
+    ======== Basic graph information ========
+    zlib/1.2.13#13c96f538b52e1600c40b88994de240f:
+      ref: zlib/1.2.13#13c96f538b52e1600c40b88994de240f
+      options:
+        fPIC: True
+        shared: False
+
+
+You can generate a graph of your dependencies, in ``dot`` or ``html`` formats:
+
+.. code-block:: bash
+
+    $ conan graph info --require=binutils/2.38 -r=conancenter --format=dot
+
+    ...
+
+    digraph {
+            "cli" -> "binutils/2.38"
+            "binutils/2.38" -> "zlib/1.2.13"
+    }
+
+
+.. graphviz::
+
+    digraph {
+            "cli" -> "binutils/2.38"
+            "binutils/2.38" -> "zlib/1.2.13"
+    }
+
+
+
+.. note::
+    The generated *index.html* output file contains links to a third-party resource, the *vis.js* library (2 files: *vis.min.js*, *vis.min.css*).
+    By default they are retrieved from Cloudfare. However, for environments without internet connection, these files
```

Comment:
I think this is also outdated. The recommendation would be to re-define the html template, but not sure it will be in the root.